src/components/useraccount/orderdetail/MainDetails.vue

```diff
@@ -0,0 +1,155 @@
+<template>
+  <div v-if="me">
+    <div class="row">
+      <div class="col-sm-6">
+        <div class="my-orders-order-dark-box">
+          <div class="my-orders-order-dark-box-title">{{ $t('shippingAddress') }}</div>
+          <div>
+            {{me.order.shippingAddress.title}}
+            {{me.order.shippingAddress.firstName}}
+            {{me.order.shippingAddress.lastName}}
+          </div>
+          <div>{{me.order.shippingAddress.streetName}} {{me.order.shippingAddress.streetNumber}}</div>
+          <div>{{me.order.shippingAddress.additionalStreetInfo}}</div>
+          <div>{{me.order.shippingAddress.city}}</div>
```

Comment:
There's a lot of repetition on these lines. Why not extract the shipping address into a computed property and access from it?